### ISSUE TITLE
BCC: AnchorUtil.CreateAnchor/CreateGridLayout are broken

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#126]
Several utility functions in AnchorUtil.lua are broken on the BCC client as the closure created for them binds to [CreateAndInitFromMixin](https://github.com/Gethe/wow-ui-source/blob/dc6f804835f7ae22bda9fceafc2feb4adf7609a3/SharedXML/Util.lua#L78-L82) which [isn't loaded until after AnchorUtil.lua](https://github.com/Gethe/wow-ui-source/blob/0bd0f71fa374bad7194f89dcb05b1c4688ac92d7/FrameXML/FrameXML.toc#L14).

The broken functions are as follows:

- [AnchorUtil.CreateAnchor](https://github.com/Gethe/wow-ui-source/blob/0bd0f71fa374bad7194f89dcb05b1c4688ac92d7/SharedXML/AnchorUtil.lua#L89)
- [AnchorUtil.CreateGridLayout](https://github.com/Gethe/wow-ui-source/blob/0bd0f71fa374bad7194f89dcb05b1c4688ac92d7/SharedXML/AnchorUtil.lua#L90)

Calling either of these functions via `/dump AnchorUtil.CreateAnchor()` or `/dump AnchorUtil.CreateGridLayout()` will result in the following error:

```text
2x SharedXML\FunctionUtil.lua:38: attempt to call upvalue 'f' (a nil value)
[string "@SharedXML\FunctionUtil.lua"]:38: in function <SharedXML\FunctionUtil.lua:38>
[string "=(tail call)"]: ?
[string "@SharedXML\Dump.lua"]:396: in function `DevTools_DumpCommand'
[string "@FrameXML\ChatFrame.lua"]:2395: in function `?'
[string "@FrameXML\ChatFrame.lua"]:4825: in function <FrameXML\ChatFrame.lua:4771>
[string "=[C]"]: in function `ChatEdit_ParseText'
[string "@FrameXML\ChatFrame.lua"]:4488: in function `ChatEdit_SendText'
[string "@FrameXML\ChatFrame.lua"]:4524: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>
```

These functions being broken impact [a few other utility functions](https://github.com/Gethe/wow-ui-source/blob/0bd0f71fa374bad7194f89dcb05b1c4688ac92d7/SharedXML/AnchorUtil.lua#L93-L94) within the same script, and also break [the grid layout for scrollbox](https://github.com/Gethe/wow-ui-source/blob/0bd0f71fa374bad7194f89dcb05b1c4688ac92d7/SharedXML/Scroll/ScrollBoxGridView.lua#L96).

**Comment by Meorawr** [comment]
Should be fixed for both clients in eventual builds - BCC will likely have it fixed a lot sooner than Classic however.

**Comment by Meorawr** [comment]
Fixed in 2.5.1.38739.
